llvm/utils/lit/tests/Inputs/shtest-cat/cat-v.txt

```diff
@@ -0,0 +1,49 @@
+# C Program to generate all ASCII characters in allchars.txt
```

Comment:
allchars is marked as a binary file in the PR. Can we either add some leading text or a file extension to make it clear to git/Github that it isn't a binary file? I know you had issues w/ `.txt` extensions getting identified as tests, but perhaps another type of extension would work? or maybe just some leading string before the strange characters?

Reply:
I believe the special characters themselves make the file binary. Even when the files used to have the .txt extension, it would still be marked as a binary file. I tried adding some leading strings but it didn't change anything.

Reply:
Interesting. If neither approach works, then there isn't much to do, I suppose. That's unfortunate, but isn't a blocker.